pandas/core/frame.py

```diff
@@ -1697,19 +1697,36 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
 
         .. versionadded:: 0.21.0
 
+        Requires either fastparquet or pyarrow libraries.
```

Comment:
Third-persion verbs should not be used, as stated in [docs guide](https://python-sprints.github.io/pandas/guide/pandas_docstring.html). There are 2 options here: either use a subject like "This function requires" or begin with the infinite "Require..."

I'm not 100% sure, but I have a feeling that "libraries" should be in singular the form "library".